conductor_api/src/interface.rs

```diff
@@ -473,14 +473,14 @@ impl ConductorApiBuilder {
                     None => None,
                 };
                 let properties = params_map.get("properties");
-                conductor_call!(|c| c.install_dna_from_file(
+                let dna_hash = conductor_call!(|c| c.install_dna_from_file(
                     PathBuf::from(path),
                     id.to_string(),
                     copy,
                     expected_hash,
                     properties
                 ))?;
-                Ok(json!({"success": true}))
```

Comment:
Envoy is actually checking for `success` to see if the request happened successfully. Maybe it's enough to check for `Ok`, i.e. not `Err`. Maybe leave `success` in until we standardize this? Thoughts from others?

Reply:
Yeah lets leave it in why not